### DISCUSSION TITLE
Geometric SMOTE for regression.

### DISCUSSION THREAD

**Discussion by Linhkust** [georgedouzas/imbalanced-learn-extra#33] [General]
I am wondering whether the contributors have the codes for G-SMOTE for regression

**Comment by georgedouzas** [comment]
Hi @Linhkust, no Geometric SMOTE for regression has not been implemented yet.

**Reply by zhouyulong2287** [reply]
I am wondering whether the contributors have the codes for G-SMOTE for regression, too. Could you tell me how to write code for G-SMOTE?
